Cargo.toml

```diff
@@ -235,7 +235,7 @@ pallet-investments = { path = "pallets/investments", default-features = false }
 pallet-keystore = { path = "pallets/keystore", default-features = false }
 pallet-liquidity-pools = { path = "pallets/liquidity-pools", default-features = false }
 pallet-liquidity-pools-gateway = { path = "pallets/liquidity-pools-gateway", default-features = false }
-pallet-liquidity-pools-gateway-queue = { path = "pallets/liquidity-pools-gateway/queue", default-features = false }
+pallet-liquidity-pools-gateway-queue = { path = "pallets/liquidity-pools-gateway-queue", default-features = false }
```

Comment:
I think we can leave this name as it is.

Reply:
But then it would look weird if the pallet is being used for another kind of queues...

Reply:
That's a problem for future-us :D Plus, I think that eventually we'll drop it and use `Tasks`.

Reply:
I'm not convinced by the name, but I will leave it as it